tree/dataframe/inc/ROOT/RDF/RSampleInfo.hxx

```diff
@@ -45,6 +49,16 @@ public:
    RSampleInfo &operator=(RSampleInfo &&) = default;
    ~RSampleInfo() = default;
 
+   std::string GetGroupName() const { return fGroupMetaData.fGroupName; }
+
+   unsigned int GetGroupID() const { return fGroupMetaData.fGroupId; }
+
+   int GetI(const std::string &cat) const { return fGroupMetaData.fMetaData.GetI(cat); }
+
+   double GetD(const std::string &cat) const { return fGroupMetaData.fMetaData.GetD(cat); }
+
+   std::string GetS(const std::string &cat) const { return fGroupMetaData.fMetaData.GetS(cat); }
+
    /// Check whether the sample name contains the given substring.
    bool Contains(std::string_view substr) const
```

Comment:
`Contains` doesn't really make sense anymore as a name, right? should probably deprecate it in favor of something more explicit